src/Switch/Switch.spec.js

```diff
@@ -41,7 +48,8 @@ describe('<Switch />', () => {
   });
 
   it('should spread custom props on the SwitchBase node', () => {
```

Comment:
I'm not sure to think about this test. I would rather see the custom properties be applied on the root node.
But that's controversial, I know @nathanmarks might not agree with this.